src/pages/video-test.tsx

```diff
@@ -33,6 +36,47 @@ export const getServerSideProps: GetServerSideProps = async function ({query}) {
   }
 }
 
+const TestSidePanel: React.FC<any> = ({className, disableCompletely}) => {
+  const {player} = usePlayer()
```

Comment:
can be arbitrary egghead-next "userland" component as long as we are in the `PlayerProvider` we can get data we need and it updates properly.